### DISCUSSION TITLE
How can i filter outbound traffic which is going from my application to outside of the servicemesh?

### DISCUSSION THREAD

**Discussion by shudhanshh** [linkerd/linkerd2#4975] [General]
I am trying to find out outbound traffic from my application/deployments to outside of my cluster/service mesh.

kubectl exec -it pod_name bash container_name
# curl google.com
# curl https://google.com
kubectl logs pod_name -c linkerd-proxy

I'm not getting the outbound logs anywhere in the deployment or sidecar proxy or debug sidecar proxy.

How can I get to see the URLs that I curled from my pod?

Thanks

**Comment by ihcsim** [comment]
The [`linkerd tap`](https://linkerd.io/2/reference/cli/tap/) command should give you those information.

**Comment by grampelberg** [comment]
Note: in your example, you're using https. As this is encrypted, the proxy does not see where the request is going. This means that it'll show up as TCP in the metrics but not in tap or any of the higher level metrics commands.
